LKMEX-whitelist/HodlTokenClub WL.md

```diff
@@ -0,0 +1,28 @@
+## Name of the project
+Hodl Token Club
+
+## Description of the project
+HodlTokenClub is an web3 organisation focused on capturing yield on xExchange and share it with its investors. We leverage MultiversX NFT's to distribute $HODL token (share of our net worth) and rewards generated on xExchange. 
+
+
+## Purpose
+We have build economic model enabling Comunity Farm growth on xExchange and generating incentive for new users to join, mint our NFT's and contributing to our ecosytem to grow as well as LKMEX rewards to grow in value as more capital is captured from minting process. Community farm is 100% own by $HODL token owners. Rewards are distributed everyday via #GiveawayEveryday scheme on Twitter. Over 4mths we distributed over 90 Million LKMEX to our Community members, eductaed them and helpped with farming on DEX. We belive we add value to entire MultiversX ecosystem as our project is solely based on capturing Yield on xExchange, growing our position and sharing part of it as rewards with community members and helping them with finding optimal strategy for their own farm. Our Yield generated in LKMEX never was sold and never will. All LKMEX rewards coming from Founder personal farm, to enable Community fam growth while capturing capital from our NFT minting process. [Why does the project need a transfer role for XMEX?]
```

Comment:
mths -> months?

Reply:
GM @iulianpascalau Yes, I shorten months. Is this a problem?

Reply:
I prefer the unshortened version. For clarity, since this is a description.

Reply:
Hi @iulianpascalau requested change have been done (I also updated description to give it more clarity and be more sound). Is there anything else that need to be done for this PR to be approved? I hope this could be done in timely manner as my project depend on it. I have around 10 really big investors seeking return from investment, they have been patient so far, but this takes really long time. I have stated this originally in November last year, and here we are in middle of March. I'm listing $HODL token weekend after next, with that listing we will use our utility token called $REWARD for farming and we will create Single Asset staking Farm. Since farming starts, we would like to add utility to reward token, but to do this, we need to launch HODL Founder NFT staking first, as Founder NFTs will be purchasable via our NFTs ONLY (Galactic Hodler Collection). Acquired NFTs then, will be purchasable via $REWARD tokens ONLY. But, to start this wheel turning we need  this PR approved and start HODL Founder NFT staking, as only then we will be in position to acquire NFTs to add utility to REWARD tokens.